src/janitor/janitor.go

```diff
@@ -147,11 +147,21 @@ func (j *Janitor) listenForNIP94Events() {
 					log.Println("eventChan closed, stopping event processing")
 					return
 				}
-				if !contains(mainConfig.TrustedMaintainers, event.PubKey) {
+				identitiesConfig := j.configManager.GetIdentities()
+				if identitiesConfig == nil {
+					log.Printf("Failed to get identities config, it is nil")
+					continue
+				}
+				trustedMaintainer, err := identitiesConfig.GetPublicIdentity("trusted_maintainer_1")
```

Comment:
Fine for now but when dynamically creating/loading a list we'll need to change this flow